packages/gasket-plugin-nextjs/generator/mocha/test/pages/index.test.js

```diff
@@ -1,9 +1,11 @@
 import { render, screen } from '@testing-library/react';
+import { IntlProvider } from '@react-intl';
```

Comment:
Is this supposed to be `react-intl`?

Reply:
Lol it should, good catch. Had Github co-pilot running as test and it seems to infer a but much 😬 (looking at the line above)